### PR TITLE
Update geofencing.yaml for release candidate 0.1.0-rc

### PR DESCRIPTION
**Pull request by akoshunyadi**
#### What type of PR is this?

* enhancement/feature

#### What this PR does / why we need it:

According to #112 

Prepared release-candidate 0.1.0-rc, including some corrections

#### Changelog input

```
- Changed version to v0.1.0-rc

```

### CODE REVIEW

**Review by bigludo7 — APPROVED** [review]
LGTM

**Review by jlurien — APPROVED** [review]

**Comment by jlurien** [comment]
Hi @akoshunyadi, please take a look into the conflicts, due to the other PR https://github.com/camaraproject/DeviceLocation/pull/115 being recently merged

**Comment by jlurien** [comment]
We can also adjust some misalignments regarding Errors:

* Generic400: Change description and message in example to: "Invalid argument"
* Generic401: Change description and message in example to: "Unauthenticated"
* Generic403: Change description to "Permission denied"

as in https://github.com/camaraproject/DeviceLocation/pull/104/files#diff-4d5f8ffcf7c3ed5fd9dcf63999a1413cba4fee213f2ae8dbf67efc460427a1ccR370-R395

**Review by jlurien — DISMISSED** [review]

**Review by bigludo7 — DISMISSED** [review]

**Comment by rartych** [comment]
Since multiple examples are not possible in Schema Object prior OAS 3.1.0 I propose to modify` Source` component definition as follows:

```
Source:
      type: string
      format: uri-reference
      minLength: 1
      description: |
        Identifies the context in which an event happened - be a non-empty `URI-reference` like:
        - URI with a DNS authority:
          * https://github.com/cloudevents
          * mailto:cncf-wg-serverless@lists.cncf.io
        - Universally-unique URN with a UUID:
          * urn:uuid:6e8bc430-9c3a-11d9-9669-0800200c9a66
        - Application-specific identifier:
          * /cloudevents/spec/pull/123
          * 1-555-123-4567
      example: "https://notificationSendServer12.supertelco.com"
```

**Review by jlurien — DISMISSED** [review]
LGTM

**Review by bigludo7 — APPROVED** [review]

**Comment by jlurien** [comment]
@akoshunyadi ,  @rartych , please tale a look to the new Device model approved by QoD: https://github.com/camaraproject/DeviceLocation/issues/117

I have already adjusted location-verification.yaml

**Comment by bigludo7** [comment]
Thanks @akoshunyadi 
2 smalls comments:

- I understood you used same description for datetime but are we sure it's not better to have explicit description for startsAt & expiresAt attribute ? Not a blocker for me but @JoachimDahlgren  point was to have explicit description.
- This description must have following sentence also: `It must follow [RFC 3339](https://datatracker.ietf.org/doc/html/rfc3339#section-5.6) and must have time zone. Recommended format is yyyy-MM-dd'T'HH:mm:ss.SSSZ (i.e. which allows 2023-07-03T14:27:08.312+02:00 or 2023-07-03T12:27:08.312Z) `in order to be aligned with the guideline (https://github.com/camaraproject/Commonalities/blob/main/documentation/API-design-guidelines.md#115-data-definitions)

**Comment by akoshunyadi** [comment]
@bigludo7 sure, I can change that, I think also other date-time fields are affected.

**Review by bigludo7 — DISMISSED** [review]
LGTM

File: code/API_definitions/geofencing.yaml [hunk 1]
```diff
@@ -148,6 +148,9 @@ paths:
           $ref: "#/components/responses/Generic500"
         "503":
           $ref: "#/components/responses/Generic503"
+      security:
+        - openId:
+            - geofencing-subscriptions:write
```
Comment (jlurien) [inline code/API_definitions/geofencing.yaml:153]:
```suggestion
            - geofencing:subscriptions:write
```
it is api-name:resource:action

File: code/API_definitions/geofencing.yaml [hunk 2]
```diff
@@ -176,19 +179,19 @@ paths:
           $ref: "#/components/responses/Generic503"
       security:
         - openId:
-            - location-retrieval:read
+            - geofencing-subscriptions:read
```
Comment (jlurien) [inline code/API_definitions/geofencing.yaml:182]:
```suggestion
            - geofencing:subscriptions:read
```

File: code/API_definitions/geofencing.yaml [hunk 3]
```diff
@@ -244,7 +247,7 @@ paths:
           $ref: "#/components/responses/Generic503"
       security:
         - openId:
-            - location-retrieval:delete
+            - geofencing-subscriptions:delete
```
Comment (jlurien) [inline code/API_definitions/geofencing.yaml:250]:
```suggestion
            - geofencing:subscriptions:delete
```

File: code/API_definitions/geofencing.yaml [hunk 4]
```diff
@@ -213,7 +216,7 @@ paths:
           $ref: "#/components/responses/Generic503"
       security:
         - openId:
-            - location-retrieval:read
+            - geofencing-subscriptions:read
```
Comment (jlurien) [inline code/API_definitions/geofencing.yaml:219]:
```suggestion
            - geofencing:subscriptions:read
```

**Review by jlurien — COMMENTED** [review]
Fix scopes naming

**Review by jlurien — APPROVED** [review]
LGTM

**Review by bigludo7 — APPROVED** [review]
LGTM

**Pull request merged**